### DISCUSSION TITLE
cn/2024/09/20/%E6%88%91%E7%9C%8B%E5%88%B0%E7%9A%84%E9%98%B6%E7%BA%A7%E5%9B%BA%E5%8C%96/

### DISCUSSION THREAD

**Discussion by giscus** [Finderaha/myblog#5] [Announcements]
# cn/2024/09/20/%E6%88%91%E7%9C%8B%E5%88%B0%E7%9A%84%E9%98%B6%E7%BA%A7%E5%9B%BA%E5%8C%96/

https://myblog-sage-seven.vercel.app/cn/2024/09/20/%E6%88%91%E7%9C%8B%E5%88%B0%E7%9A%84%E9%98%B6%E7%BA%A7%E5%9B%BA%E5%8C%96/

**Comment by hongtaoh** [comment]
我最近也在思考这个问题。我的感觉是学习的效用确实在降低。普通人现在读大学、研究生最多能让自己比父辈好一些，但是天花板是常见的。如果国企、事业单位这种固化很牢固，那只有卷学术和商业这两条路。卷学术就算进一个 985 当教授，按当前的社会标准来看也不至于说完成了阶层跃迁。那商业的话，真正创业成功的人寥寥可数。感觉没什么路了。或者出国？现代社会要学的东西太多了。在这个时代想，我觉得想通过学习快速变现是很难。看来，最后只能看命了，哈哈。
